gcc/testsuite/rust/compile/bad-crate-name.rs

```diff
@@ -0,0 +1,4 @@
+// { dg-additional-options "-frust-crate=bad+name" }
+// { dg-excess-errors "invalid crate name: ...." }
+// { dg-excess-errors "unrecognized command-line option ...." }
```

Comment:
Is there a better way to write those directives? It looks like error messages w/o location information can't be matched by `dg-error`.

Reply:
@dkm ^^ is this the best way? I think it is i cant think of how we handle error messages without location info.